maintenance_equipment_sequence/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 ForgeFlow S.L. (https://forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+def migrate(cr, version):
+    """
+    Column `code` of table `maintenance_equipment` has been modified to not have duplicates
+    and then has been copied to `serial_no` column because they have the same function and
+    `serial_no` is the one in the core.
+    Afterwards, column 'code' has been deleted.
+    """
+
+    if not version:
+        return
+
```

Comment:
add
```suggestion

    cr.execute(
        """
        UPDATE maintenance_equipment
        SET code = serial_no
        WHERE code IS NULL;"""
    )

```
in order to avoid unexpected rare case:
```
coming with:

a: id=1, code=NULL, serial_nu=5
b: id=2, code=5, serial_nu=XX

and you end with:

a: id=1, serial_nu=5
b: id=2, serial_nu=5
```